integration-tests/manifests/sample-statefulset.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: my-statefulset
```

Comment:
Can we create a sample-namespace file as well and keep these resources in that namespace so they dont just end up in default?
We need to test scenarios that auto annotate a whole namespace as well.

Reply:
We are testing the default namespace, but I can create sample-namespace, but you want me to put all the other sample apps in this one new namespace?

Reply:
Left another comment - I think it would be better to have a sample-namespace for some namespace related usecases but for most others, create a `sample-namespace-<hash>` on the fly so we isolate the test cases from one another.

Be sure to clean up resources after each test case i.e. delete the namespace once its done.

Reply:
Okay will do, also we don't need a whole different operator manager for each right? I just put a lock around when the operator is used

Reply:
yeah just 1 operator for the entire cluster -- and workloads in a sample namespace.